tests/jobs/test_backfill_job.py

```diff
@@ -52,6 +52,7 @@
 DEFAULT_DATE = timezone.datetime(2016, 1, 1)
 
 
+@pytest.mark.quarantined
 class TestBackfillJob(unittest.TestCase):
```

Comment:
I dunno... it seems to impose a lot of risk to skip **this** class

Reply:
Sure. I will create an issue assign you to it firs :)

Reply:
But on a more serious note - > our goal is to clean up everything from Quarantine before the release. I think for master we can afford some temporary lack of testing.

Side note - the way we have the tests now - they are skipping all Py3.7 tests :(. And we are merging failing tests anyway. I think we should simply - once we get stable have a project where we have all classes in Quarantine, assign people and make them move out of quarantine.

Reply:
I know. I just wanted to express my concerns. Let's move on and fix those tests asap